seedfarmer/services/_service_utils.py

```diff
@@ -38,34 +33,72 @@ def get_botocore_config() -> botocore.config.Config:
     )
 
 
-def boto3_client(service_name: str) -> boto3.client:
-    return boto3.Session().client(service_name=service_name, use_ssl=True, config=get_botocore_config())
+def create_new_session(region_name: Optional[str] = None, profile: Optional[str] = None) -> Session:
+    return Session(region_name=region_name if region_name else None, profile_name=profile if profile else None)
```

Comment:
you don't need the `if .. else ..` around each of these. region_name is either a string or None, profile is either a string or None.either can be passed to Session as is

Reply:
I will evaluate and remediate

Reply:
complete